### ISSUE TITLE
Should we make institution.type required?

### ISSUE THREAD

**Issue by MortenHofft** [gbif/registry#84]
Currently `institution.type` is an optional enum.
http://api.gbif.org/v1/grbio/institution/bb75864c-c293-4a18-b949-71ce436d0a13

Should it perhaps be required? Especially since we have the enum `OTHER_INSTITUTIONAL_TYPE`

See all here: http://api.gbif-dev.org/v1/enumeration/basic/InstitutionType

**Comment by asturcon** [comment]
In the data we migrated from GRBIO there are a lot of records with the type null. I don't know if the `OTHER_INSTITUTIONAL_TYPE` enum can be applied to all these null values or not. We should wait till the data is cleaned.

**Comment by ManonGros** [comment]
yes please, we can try to populate the field from other information in the records but I don;t think we can make it mandatory
